### ISSUE TITLE
Using MO2a3 to install homemake Expanded Settlements crashes game of start

### ISSUE THREAD

**Issue by SonicBog** [TanninOne/modorganizer#491]
Just tried to install the latest release of Homemaker Expanded is Settlement keywords and as soon as I start the game (or start building a settlement cannot remember which) it crashes.
I reverted to using NMM and it installs fine and runs fine.

Just install Settlement Keywords 0.5 and Hommaker Expanded Settlements 1.3 and run. BOOM.

Regards
Darren

**Comment by QuietusPlus** [comment]
Having the same issue (running alpha2, newer one crashes).

It seems ModOrganizer isn't loading meshes properly, maybe due to amount of files? Issue also happens when you manually extract the files to /Data (i.e. unmanaged).

**Comment by GSDFan** [comment]
This is most likely a mod problem with those mods and not MO. I did the same thing, update those mods, and had problems with the game. After a few hours of disabling mods I got the game to run. A few of the patches I had for mods were the problem.

**Comment by Rifmaz666** [comment]
There is a limit on how many categories and Items you can have in the Settlement Building screen. I was having problems as well until I eliminated a couple of mods and changed the load order.

**Comment by SonicBog** [comment]
Just wanted to let you know that there is absolutely no problem with the mod.
It installs perfectly fine using NexusModManager.
It has been updated to use Settlement keywords which reduces the number of categories.
Also, going past the limit does not cause crashes, it just means you end up missing items.

**Comment by michaelrw10** [comment]
yes this is very unlikely caused by a mod. I am experiencing the same crash after start, have no mods activated in MO, only have some loose files in the actual data directory (cbbe, bodyslide, a few texture mods, etc. -- no plugins active) .. crash every time i try to launch FO4 from MO.. launching from outside of MO is fine, and has been fine for the past few months. It is very clear that MO is not handling something correctly.

**Comment by QuietusPlus** [comment]
Definitely not the mod at fault here, works perfectly fine using NMM.

This is the same behaviour I've seen when I forget to add a mesh, always crashes upon opening the workbench. ModOrganizer isn't loading them properly.

You can recreate this by simply deleting a mesh used for constructable objects.

**Comment by AnyOldName3** [comment]
I updated homemaker yesterday, and it was crashing. When I investigated, MO had installed the meshes to \data\data\meshes instead of \data\meshes, so it obviously didn't work. Either the mod has a bad install script, which NMM happens to be correcting, or MO isn't installing things properly.

**Comment by Rifmaz666** [comment]
Running Windows 7 Pro x64 32GB Ram Dual Hex- Core Xeon Processors

I was crashing as well but removed the Buildable Elevators and Greenhouse mods and the game is launching fine for me now. MO does crash when installing any Mods but it does successfully install them.

**Comment by QuietusPlus** [comment]
@AnyOldName3

That doesn't explain manually installing and it still crashing though. Try extracting the files to /Data and activating it (would still crash in my case).

I've had mixed experiences with scripted installers before, works 90 percent of the time but sometimes breaks where NMM doesn't.

Have switched to NMM in the mean time but will try this myself soon. I miss ModOrganizer already, alpha2 was working fine before this, first major issue encountered (apart from a3 not launching).

**Comment by TheGreenDragon** [comment]
Just installed Homemaker 1.31d with MO 2a3 and can confirm that MO's automatic install copied all the files correctly but moved meshes and textures files to Data\Data\Meshes and Data\Data\Textures for me. After manually moving them to the correct location game and mod both loaded fine. Installed the SK Compatibilty Patch and Unlocked Institute Objects optional ESP's with the base mod.

**Comment by Sir1Twist** [comment]
I do believe it is a bad install script for the mod over anything else. I've had the same problem with it. I'm deciding not use Homemaker until the construction kit comes out, and then trying it at that point. Maybe all the mods will be really fixed. I hate having to run mods and patches, makes things untidy.

**Comment by epochfx** [comment]
Here's a workaround.  The scripted installer works with NMM but does not work with Mod Organizer for whatever reason.

1. Extracting the mod .zip
2. Moving the optional .esps I need into the mods data folder
3. Re-zipping the mod's data folder
4. Installing the edited .zip using the mod manager's install button
